contents/handbook/people/hiring-process/index.mdx

```diff
@@ -300,16 +300,25 @@ As a rule, all interviews at PostHog are conducted in English. Whilst this might
 
 If you are paired with an interviewee who speaks your native language, just politely acknowledge this and let them know all interviews are conducted in English. We also require these calls to be conducted as a video call, so a working webcam is necessary.
 
-### How interviews are scored
+### How we score interview candidates
 
-Scoring Scale (1– 4)
+We score candidates from 1 to 4, 1 being a strong no and 4 a strong yes:
 
--   1: Strong No = This candidate is clearly not a fit for PostHog now or in the future
--   2: No = Not a fit now (maybe in the future)
--   3: Yes = This is a solid hire
--   4: Strong Yes = This is an exceptional person we need to hire. (we might go to extra lengths to hire them)
+- **1. Strong No = Never:** Applies to anyone who you don't think could ever raise the bar here. Obviously bad culture fit is an automatic strong no, even if they have the relevant skills for the role. 
 
-> A good rule of thumb when deciding whether not to progress at any stage: if the candidate is between a 2 and a 3, then it's a 2. It's almost never worth putting through someone who is a 'maybe'! We provide lots of information about PostHog to enable candidates to put their best application forward.
+- **2. No = Not now** Only applies to candidates who you would *confidently* consider in future. This normally means they're a good culture fit who could raise the bar, but they lack something we specifically need in the role we're considering them for right now.
+
+- **3. Yes = Could raise the bar:** Applies to anyone who you think could raise the bar and fits our needs, but you're not 100% sure. Explain your mild concerns and what it would take for you to be convinced.  
+
+- **4. Strong yes = Very likely to raise the bar:** Applies to anyone you're **very confident** will raise the bar and/or has a high ceiling. We might go to extra lengths to hire them.
+
+### Useful scoring principles and tips
+
+- We should only hire people who we're confident will either raise the bar immediately, or who we think have a high ceiling and are likely to do so in the foreseeable future.
```

Comment:
I agree with everything here, but this is the bit where I think we do and say different things - I think I and probably most people hire people who _meet_ the bar, ie. are at least as good as the existing team. But I don't think we _exclusively_ hire people who are better than everyone else on the team. 

I actually think this is ok, and wonder if 'raise the bar' is just not going to happen 100% of the time. We've hired lots of people who 'only' meet the bar who then go on to do amazing work. Maybe it's 'we can see them realistically raising the bar' rather than 'they will obviously raise the bar from day 1'. Or maybe I'm doing hiring wrong (but I think most/all of us actually do this in practice).